tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/passes/BoundedCheckerPassImpl.scala

```diff
@@ -75,7 +76,12 @@ class BoundedCheckerPassImpl @Inject() (
 
     val smtProfile = options.common.smtprof
     val smtRandomSeed = tuning.getOrElse("smt.randomSeed", "0").toInt
-    val solverConfig = SolverConfig(debug, smtProfile, smtRandomSeed, smtEncoding)
+    // Parse the tuning parameters that are relevant to Z3. Some passes add more fields, which cannot be parsed.
```

Comment:
What does this mean: `Some passes add more fields, which cannot be parsed.` ?

Reply:
What I noticed is that `tuning` as a map contains some options that were not present in the passed options file. I am not sure how exactly configuration options are processed. Still on my todo list.